src/frontend/magnify/src/views/rooms/list/RoomsListView.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { RoomsListView } from './index';
+
+export default {
+  title: 'Views/Rooms/RoomsView',
+  component: RoomsListView,
+} as ComponentMeta<typeof RoomsListView>;
+
+const Template: ComponentStory<typeof RoomsListView> = (args) => {
+  return (
+    <RouterProvider
```

Comment:
ditto

Reply:
Yes for the moment the package is too linked to the router. Gotta undo it